Client/game_sa/CCarEnterExitSA.h

```diff
@@ -17,12 +17,14 @@
 #define FUNC_GetNearestCarPassengerDoor         0x650BB0
 #define FUNC_ComputeTargetDoorToExit            0x64F110
 #define FUNC_IsRoomForPedToLeaveCar             0x6504C0
+#define FUNC_GetPositionToOpenCarDoor           0x64E740
 
 class CCarEnterExitSA : public CCarEnterExit
 {
 public:
     bool GetNearestCarDoor(CPed* pPed, CVehicle* pVehicle, CVector* pVector, int* pDoor);
     bool GetNearestCarPassengerDoor(CPed* pPed, CVehicle* pVehicle, CVector* pVector, int* pDoor, bool bUnknown, bool bUnknown2, bool bCheckIfRoomToGetIn);
+    void GetPositionToOpenCarDoor(CVector& position, CVehicle* vehicle, unsigned int door);
```

Comment:
Can const qualifier be applied here?

Reply:
On CVehicle? Not quite, it gets casted to CVehicleSA.

Reply:
On the function itself